.github/workflows/publish-preview.yml

```diff
@@ -0,0 +1,66 @@
+name: Publish a preview build
```

Comment:
`Publish a preview build` currently doesn't work, but we'd like to utilize the ability to publish package previews. A recent example is the brand evolution work for the design tokens and publishing builds like in the portfolio. I've created an issue here: https://github.com/MetaMask/metamask-design-system/issues/26.

Reply:
Add a comment to the file that this script isn't working and to add a link to the issue "To do:"